lib/media/track/remotetrackpublication.js

```diff
@@ -70,6 +70,23 @@ class RemoteTrackPublication extends TrackPublication {
       }
       this.emit(signaling.isEnabled ? 'trackEnabled' : 'trackDisabled');
     });
+
+    signaling.on('trackswitchedon', () => {
```

Comment:
@makarandp0 We typically do not use multiple events for updates to different signaling parameters. Can we just use the "updated" event instead?

Reply:
currently `RemoteTrackPublicationSignaling` fires  `updated` , and `RemoteTrackPublication` fires trackEnabled/trackDisabled based on it. 

if we updated `RemoteTrackPublicationSignaling` to fire `updated` for track switch on/off as well, then it introduces ambiguity on `RemoteTrackPublication` side, which has to determine if it was track enable/disable or swich on/off. To avoid adding a state variable, I instead preferred separate event for switch on/off.